go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -351,14 +350,14 @@ func (c *Controller) syncServices(key string) error {
 
 			// Node Port
 			if svcPort.NodePort != 0 {
-				// We need to have only local endpoints on GR here, so repass all endpoint slices so local ones can be found
+				// Ensure we only add correct eps to VIP per node
```

Comment:
If you are fixing unit tests then it should be in the corresponding commit that broke the unit test